packages/core/scripts/severless-cloud-run/CloudRunHub.js

```diff
@@ -96,26 +96,26 @@ hub.post("/", async (req, res) => {
     });
 
     // Validate that the promises returned correctly. If ANY have error, then catch them and throw them all together.
-    let errorOutputs = [];
-    let validOutputs = [];
+    let errorOutputs = {};
+    let validOutputs = {};
     results.forEach((result, index) => {
-      if (result.status == "rejected" || result?.value?.execResponse?.error || result?.reason?.code != "200") {
+      if (result.status == "rejected" || result?.value?.execResponse?.error || result?.reason?.code == "500") {
```

Comment:
I reverted this back to explicitly checking for code 500 as the spoke always errors with this code. checking `!="200"` would incorrectly detect some situations as errors as a status code is not always sent.

Reply:
You might want to switch this to only checking if the code starts with "2"?